### ISSUE TITLE
Razer Firefly v2

### ISSUE THREAD

**Issue by wEbAddEr** [openrazer/openrazer#1190]
The Razer Fierfly v2 is not detected.
Still not detected after the update with the command prompt.

**Comment by z3ntu** [comment]
Can you please share the output of the command `sudo dkms install openrazer-driver/2.8.0` ? Also have you looked at the [help page for Troubleshooting](https://github.com/openrazer/openrazer/wiki/Troubleshooting) at all?

**Comment by KayJay5** [comment]
I've got this issue also.

```
:~$ sudo dkms install openrazer-driver/2.8.0
Module openrazer-driver/2.8.0 already installed on kernel 4.15.0-111-generic/x86_64
```

My firefly is being picked up as:
`Bus 003 Device 029: ID 1532:0c04 Razer USA, Ltd `

`2020-07-22 06:09:14 | razer                          | DEBUG    | Loaded device specification: RazerFirefly ---------------------------- (1532:0C00)`

I've gone through the troubleshooting section and everything seems to be working.  I  have a razer mamba and a goliathus extended that are both working.

I've got a Cynosa Chroma Pro that is also not being detected.

`Bus 003 Device 004: ID 1532:022c Razer USA, Ltd `

`2020-07-22 06:09:14 | razer                          | DEBUG    | Loaded device specification: RazerCynosaChroma ----------------------- (1532:022A)`


Thanks for your help.
